File: robocode2-bot-api-java/src/main/java/net/robocode2/events/ConnectedEvent.java
package net.robocode2.events;

import lombok.Builder;
import lombok.ToString;
import lombok.Value;

/** Event occurring when bot gets connected to server */
@Value
@Builder
@ToString
public class ConnectedEvent {

  /** HTTP status code */
  short httpStatus;

  /** Status message */
  String httpStatusMessage;
}

File: robocode-tankroyale-bot-api-csharp/src/events/ConnectedEvent.cs
using Newtonsoft.Json;

namespace Robocode.TankRoyale.BotApi
{
  /// <summary>
  /// Event occurring when bot gets connected to server.
  /// </summary>
  public sealed class ConnectedEvent
  {
    // Currently empty. Preserved for future

    /// <summary>
    /// Initializes a new instance of the ConnectedEvent class.
    /// </summary>
    [JsonConstructor]
    public ConnectedEvent() { }
  }
}